### PR TITLE
chore(scrollbar): added new scrollbar to emote

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨
AP-1481

**Special notes for reviewers** 🗒️

**Additional comments** 🎤
New scrollbar component's first add, can just be found on the emoji picker

### CODE REVIEW

File: assets/styles/base.less [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :root {
   --app-height: 100vh;
-  --flair-color: @primary-color;
+  --flair-color: '#2761fd';
```
Comment (molimauro) [inline assets/styles/base.less:3]: @ThomBos why arent you using the @primary-color var?

File: components/views/chat/enhancers/Enhancers.vue [hunk 2]
```diff
@@ -85,12 +90,19 @@ export default Vue.extend({
...
           // @ts-ignore
-          if (!this.$refs.emojiPicker) this.$refs.emojiInvoker?.click()
+          if (!this.$refs.emojiPicker) {
+            if (this.$refs.emojiInvoker) {
+              this.$refs.emojiInvoker.click()
+              return
+            }
+            setTimeout(() => {
+              this.$refs.emojiInvoker?.click()
+            }, 200)
```
Comment (molimauro) [inline components/views/chat/enhancers/Enhancers.vue:104]: Why a setTimeout nested in another setTimeout?

Comment (ThomBos) [reply components/views/chat/enhancers/Enhancers.vue:104]: because we haven't the component yet, even if openEmoji() is called on mounted

**Review by molimauro — APPROVED** [review]

**Review by Clayton215 — APPROVED** [review]

**Review by stavares843 — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/29093946/166314116-a4e98f4e-14ec-4233-a488-32fd0e21a1af.mov

seems that when switching from the glyphs tab to the emojis tab dev vs this branch, seems this branch is slower

would this be related to the new scroll?

browser - safari

btw is pretty neat on mobile 🎉

**Comment by ThomBos** [comment]
> seems that when switching from the glyphs tab to the emojis tab dev vs this branch, seems this branch is slower


this issue it's going to be solved on another branch, i'm gonna open the PR tomorrow 👍 🎉

**Comment by stavares843** [comment]
thanks for clarifying 🔨

**Comment by ThomBos** [comment]
This PR depends on https://github.com/Satellite-im/Core-PWA/pull/3042 💥

**Comment by stavares843** [comment]
https://github.com/Satellite-im/Core-PWA/pull/3042 merged 💥 

missing fixing conflict 🔨

**Review by phillsatellite — APPROVED** [review]

**Comment by molimauro** [comment]
It's so good to see all those enhancers tricks gone. Thanks @ThomBos

**Review by molimauro — APPROVED** [review]

**Pull request merged**